src/cf-forms/src/cf-forms.less

```diff
@@ -219,6 +219,340 @@ textarea {
+        box-shadow: 0 0 0 1px @input-bg, 0 0 0 2px @input-border;
+    }
+
+    &[type="checkbox"]:checked + label::before {
+        content: @cf-icon-approved;
+        font-family: 'CFPB Minicons';
+        font-size: unit( 16px / @base-font-size-px, em);
+        box-shadow: 0 0 0 1px @input-bg, 0 0 0 2px @input-active;
+        text-align: center;
+    }
+
+    &[type="checkbox"]:hover + label::before {
+        box-shadow: 0 0 0 1px @input-bg, 0 0 0 2px @input-active;
+    }
+
+    &[type="checkbox"]:focus + label::before {
+        box-shadow: 0 0 0 2px @input-active;
+    }
+
+    &[type="checkbox"]:disabled + label {
+        color: @input-border-disabled;
+    }
+
+    &[type="checkbox"]:disabled + label::before,
+    &[type="radio"]:disabled + label::before {
+      background: @input-bg-disabled;
+      box-shadow: 0 0 0 1px @input-bg, 0 0 0 2px @input-border-disabled;
+      cursor: not-allowed;
+    }
+
+}
+
+//
+// Form layouts
 //
+
+.form-l-inset {
+    border-right-width: 2.5px;
+
+    + .form-l_col-1-2.form-l-inset:nth-child(odd) { 
+        border-left-width: 2.5px;
+    }
+    
+    &.form-l_col-1-2:nth-child(odd),
+    &.form-l_col-1-2:last-child {
+        border-right-width: 15px;
+    }
+
+    .form-l-inset_container {
+        background: @input-inset-bg;
+
+        .cf-input_label {
+            margin: 0;
+            padding: unit( ( @grid_gutter-width / 2 ) / @base-font-size-px, em )
+               unit( ( @grid_gutter-width / 2 ) / @base-font-size-px, em )
+               unit( ( @grid_gutter-width / 1.75 ) / @base-font-size-px, em );
+        }
+
+        .cf-input_text {
+            margin-left: unit( @grid_gutter-width / @base-font-size-px, em );
+        }
+
+        .cf-input_radio,
+        .cf-input_checkbox {
+            position: absolute;
+            top: unit( ( @grid_gutter-width / 1.75 ) / @base-font-size-px, em );
+            left: 15px;
+            z-index: 3;
+        }
+
+        &:hover,
+        &:focus,
+        &.focus {
+            .cf-input_checkbox,
+            .cf-input_radio {
+                box-shadow: 0px 0px 0px 2px @input-border-focus;
+                
+                // Older IE does not support border-radius or box-shadow it's ok to
+                // use outline here.
+                .lt-ie9 & {
+                    outline: 1px solid @input-border-focus;
+                }
+            }
+        }
+
+        &.selected .cf-input_label,
+        &.focus .cf-input_label,
+        &:focus .cf-input_label,
+        &:hover .cf-input_label,
+        input:focus + .cf-input_label,
+        input:checked + .cf-input_label     {
+            outline: 2px solid @input-border-focus;
+        }
+
+        &.selected .cf-input_label,
+        input:checked + .cf-input_label {
+            background-color: @input-inset-selected;
+        }
+    }
+}
+
+//
+// m-select menu
+//
+
+.m-select {
+    position: relative;
+    margin-top: 0.5em;
+    border: 1px solid @gray-40;
```

Comment:
just a note that i'm adding a new variable for the select-border since its unique, will update cf-generator to reflect